src/constants/localeSupport.js

```diff
@@ -0,0 +1,4 @@
+export const locale = {
```

Comment:
These i18n strings should be in the locals directory.

Reply:
If we are implementing the method you mentioned in the above comment ,we can remove this file.